spec/requests/api/v1/users_spec.rb

```diff
@@ -4,8 +4,28 @@
   let(:user) { create(:user) }
   let(:Authorization) { "Bearer #{generate_jwt_token(user)}" }
 
-  path '/api/v1/users' do
+  path '/api/v1/users/all' do
```

Comment:
Need to remove 'all' from the route.rb and here, cause this CRUD method (index)

Reply:
skip